browser/src/control/Control.FormulaBar.js

```diff
@@ -139,7 +139,7 @@ L.Map.include({
 			target = mobileTopBar;
 		} else {
 			jsdialogFormulabar.hide('startformula');
-			jsdialogFormulabar.hide('AutoSumMenu');
+			$('#AutoSumMenu').hide();
```

Comment:
not happy with this... would prefer to re-use the hide function but then again I don't know why we have in the same file jquery methods and then this. Nevertheless using out custom hide and place there the uno command didn't work

Reply:
it doesn't work due to my commit:
commit a7ece5d0d615ca1857b4d020844bfa098cf9fb07
jsdialog: reduce warnings in console for formulabar

I thought it is unwanted core message...